### ISSUE TITLE
Website scraper in python2 or 3

### ISSUE THREAD

**Issue by drekhi12** [drekhi12/Scripts#11]

**Comment by Evantm** [comment]
I can do this. What websites would you like it to scrape?

**Comment by dheiberg** [comment]
I'm happy to help aswell if the task is bigger than just a single scraper

**Comment by drekhi12** [comment]
The script should be able to scrape the whole website.(Maybe use Scrapy if you can do it without that its fine too)

**Comment by dheiberg** [comment]
But which website would you like to scrape? and for what content?

**Comment by drekhi12** [comment]
It's more like making a script which could mirror the whole website. Basically, the front end has to be scraped and the website based on normal HTML CSS is our main target. Hope I am clear this time. If any queries feel free to shoot a message here.

**Comment by managedkaos** [comment]
@drekhi12 if you want to do a mirror would a `wget` solution be more applicable vs a python script to scrape?  I think the Q's here are asking if there is any specific data that needs to be scraped and compiled instead of mirroring the site into HTML.
You could do something like this:
```
wget --mirror --convert-links --adjust-extension --page-requisites --no-parent http://example.org
```
Also, if you have the URL for the site, that would be helpful.

**Comment by dheiberg** [comment]
I think he means a general site copy script. But yes, you can already do what you are talking about using wget as mentioned, if we understand you correctly

**Comment by drekhi12** [comment]
@managedkaos I agree with you but I was looking forward to making it in python using scrapy or any other library. If you want you can put that in the shell script and push that.

**Comment by Evantm** [comment]
I have made a pull request that does what wget does but in Python

**Comment by managedkaos** [comment]
PR submitted for the shell-based solution. ^_^

**Comment by drekhi12** [comment]
Hey, Nice work everyone if you like you can contribute to Scripts Repo and submit some scripts of your choice. If you have any suggestions/advice do keep this thread running.
Thanks
